src/pages/Home.tsx

```diff
@@ -1,7 +1,28 @@
+import { usePublicSets } from '@/queries/usePublicSets';
+import { Set } from '@/queries/types';
+
+function ViewSet({ set }: { set: Set }) {
```

Comment:
Move this component to a separate file

Reply:
It's better to pass the props as not an object, but just the values of it so for example:
```suggestion
function ViewSet(set: Set) {
```

and then in the `Home` component pass them like this 
```
<ViewSet key={set.id} {...set} />
```
or 
```
 {data?.map(({ title, author, terms }) => 
        <ViewSet
          key={set.id}
          title={title}
          author={author}
          terms={terms}
        />
 )}
```

As with current `<ViewSet set={set} />` you are passing a new object all the time and the code may be harder to optimize if needed also it's more understandable what props the component needs.

You can also make one additional change as the `<ViewSet />` component only uses the `length` of terms, so you could just pass 
```
 {data?.map(({ title, author, terms }) => 
        <ViewSet
          key={set.id}
          title={title}
          author={author}
          termsCount={terms.length}
        />
 )}
```
so you aren't passing a whole array with potentially a big amount of data, and in the future, if you want to optimize the component by `React.memo` it will be easier to do so, as you won't be passing a new array each time (`[]===[]` gives `false`, but with count `55===55` gives `true` so it's easier to track if the props actually changed)